src/main/java/com/futurecollars/OOPTask3/Manager.java

```diff
@@ -0,0 +1,31 @@
+package com.futurecollars.OOPTask3;
+
+public class Manager extends BaseEmployee {
+
+    public static final int MANAGER_BASE_SALARY = 5000;
+    public static int BONUS = 500;
```

Comment:
to też nie jest stała (możesz ją zmienić za pomocą setBonus) więc CZEMU_WYGLADA_JAK_STALA? 

+ nie powinna być static.

Reply:
czym się różni stala od zwyklej zmiennej - stala **zawsze** jest taka sama.

np. Math.PI to zawsze 3,14.....

BONUS natomiast tutaj jest 500, ale możemy równie dobrze zmienić na 600 (za pomocą setBonus) i już powoduje to, że to nie jest stala.